### PR TITLE
Feature/update Backbone documentation

### PR DESCRIPTION
**Pull request by tnotheis**

### CODE REVIEW

File: _docs_explore/51-backbone.md
Comment (jkoenig134) [inline _docs_explore/51-backbone.md:18]: Event Handler is missing

Comment (tnotheis) [reply _docs_explore/51-backbone.md:18]: Doesn't it? 
Maybe we should define who the reader of this document is. 😉 For me it is the person hosting the Backbone. Because who else would be interested which infrastructure exists. 
And if my assumption is correct, then this person would also be interested in the existence of the Admin UI - at least in my opinion.

Comment (jkoenig134) [reply _docs_explore/51-backbone.md:18]: Ok, makes sense.

**Review by jkoenig134 — DISMISSED** [review]

**Comment by tnotheis** [comment]
I'm gonna wait for approval from @stnmtz.

**Comment by tnotheis** [comment]
@stnmtz Remember to look over this PR.

**Review by jkoenig134 — DISMISSED** [review]

**Review by jkoenig134 — APPROVED** [review]

**Pull request merged**
